src/components/common/ExternalLink/index.tsx

```diff
@@ -9,17 +9,25 @@ import { Box, Link, type LinkProps } from '@mui/material'
 const ExternalLink = ({
   noIcon = false,
   children,
+  href,
+  mode = 'link',
   ...props
-}: Omit<LinkProps, 'target' | 'rel'> & { noIcon?: boolean }): ReactElement => {
-  if (!props.href) return <>{children}</>
-
-  return (
+}: Omit<LinkProps, 'target' | 'rel'> & { noIcon?: boolean; mode?: 'button' | 'link' }): ReactElement => {
+  if (!href) return <>{children}</>
+  return mode === 'link' ? (
     <Link rel="noreferrer noopener" target="_blank" {...props}>
       <Box display="inline-flex" alignItems="center" gap={0.2} component="span">
         {children}
         {!noIcon && <OpenInNewRounded fontSize="small" />}
       </Box>
     </Link>
+  ) : (
+    <Button variant="outlined" href={href} rel="noreferrer noopener" target="_blank" sx={props.sx}>
+      <Box display="inline-flex" alignItems="center" gap={0.2} component="span">
+        {children}
+        {!noIcon && <OpenInNewRounded fontSize="small" />}
+      </Box>
```

Comment:
Nit: this duplicates the `children` of the above `Link`. Let's move it to a variable and use it inside both.